examples/graph3d/16_styling_surface.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
```

Comment:
Could you run this file through Prettier, please?

PS: `npx prettier --write examples/graph3d/16_styling_surface.html`